packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -109,6 +124,68 @@ describe('Withdraw ERC20 Token', () => {
       })
     })
 
+    it('should claim funds', { defaultCommandTimeout: 200_000 }, () => {
+      // increase the timeout for this test as claim button can take ~(20 blocks *10 blocks/sec) to activate
+
+      cy.login({ networkType: 'parentChain' }) // login to L1 to claim the funds (otherwise would need to change network after clicking on claim)
+
+      cy.findByLabelText('Open Transaction History')
+        .should('be.visible')
+        .click()
+
+      cy.waitUntil(
+        () =>
+          cy
+            .findByLabelText(
```

Comment:
#Maybe we can extract this button to commands as it's used multiple times. Something like `findClaimButton`

Reply:
Done `findClaimButton`